src/awkward/operations/ak_to_torch.py

```diff
@@ -0,0 +1,67 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward/blob/main/LICENSE
+
+from __future__ import annotations
+
+import awkward as ak
+from awkward._dispatch import high_level_function
+from awkward._nplikes.numpy_like import NumpyMetadata
+
+__all__ = ("to_torch",)
+
+np = NumpyMetadata.instance()
+
+
+@high_level_function()
+def to_torch(array):
+    """
+    Args:
+        array: Array-like data. May be a high level #ak.Array,
+            or low-level #ak.contents.ListOffsetArray, #ak.contents.ListArray,
+            #ak.contents.RegularArray, #ak.contents.NumpyArray
+
+    Converts `array` (only ListOffsetArray, ListArray, RegularArray and NumpyArray data types supported)
+    into a PyTorch Tensor, if possible.
+
+    If `array` contains any other data types (RecordArray for example) the function raises a TypeError.
+    """
+
+    # Dispatch
+    yield (array,)
+
+    # Implementation
+    return _impl(array)
+
+
+def _impl(array):
+    try:
+        import torch
+    except ImportError as err:
+        raise ImportError(
+            """to use ak.to_torch, you must install 'torch' package with:
+
+        pip install torch or conda install pytorch"""
```

Comment:
This makes it possible to copy-paste the line from the error message onto the terminal. (On most terminals, triple-click selects the whole line, and leading spaces are irrelevant to shell commands.)

```suggestion
        pip install torch

or

        conda install pytorch"""
```